src/neo/Cryptography/Crypto.cs

```diff
@@ -59,39 +62,42 @@ public static byte[] Sign(byte[] message, byte[] prikey, byte[] pubkey)
         /// <returns><see langword="true"/> if the signature is valid; otherwise, <see langword="false"/>.</returns>
         public static bool VerifySignature(ReadOnlySpan<byte> message, ReadOnlySpan<byte> signature, ECC.ECPoint pubkey)
         {
-#if OSX
-            try
+            if (IsOSX)
```

Comment:
suggest `if (IsOSX && pubkey.Curve != ECC.ECCurve.Secp256r1)` instead as OSX has `Secp256r1` supportting

Reply:
.NET already has support for Secp256r1. Why switch to a 3rd party library (even one as good as BC) for something .NET has built in?

Reply:
@devhawk  it does not support secp256k1 on OSX platform.

Reply:
My bad - didnt read the original bug